examples/eks-cluster-with-observability/README.md

```diff
@@ -0,0 +1,146 @@
+# EKS Cluster with Observability Tools
+
+This example deploy a new Kubernetes Cluster with Amazon Managed Prometheus, Amazon Managed Grafana, FluentBit, and Opensearch. It also includes instructions for deploying a sample workload with ArgoCD to generate logs and metrics.
+
+---
+**NOTE**
+
+For the sake of simplicity in this example, we store sensitive information and credentials in `dev.tfvars`. This should not be done in a production environment. Instead, store the information in AWS Secrets Manager.
+
+---
+
+## How to Deploy
+
+### Prerequisites
+
+- An existing Amazon Managed Grafana (AMG) Workspace.
+  - As of this writing (January 25, 2022), the AWS Terraform Provider does not support Amazon Managed Grafana, so it must be manually created beforehand. Instructions [here](https://docs.aws.amazon.com/grafana/latest/userguide/getting-started-with-AMG.html)
+
+#### Generate a Grafana API Key
+- Give the SSO user you set up in when creating the AMG Workspace admin access. 
+  - In the AWS Console, navigate to Amazon Grafana. In the left navigation bar, click __All workspaces__, then click __eks-ssp-observability__
+  - Under __Authentication__ within __AWS Single Sign-On (SSO)__, click __Configure users and user groups__
+  - Check the box next to the SSO user you created and click __Make admin__
+- Navigate back to the Grafana Dashboard. If you don't see the gear icon in the left navigation bar, log out and log back in.
+- Click on the gear icon, then click on the __API keys__ tab.
+- Click __Add API key__, fill in the _Key name_ field and select _Admin_ as the Role.
+- Copy your API key into `dev.tfvars` under `grafana_api_key` 
+
+### Deployment Steps
+
+- Clone this repository: `git clone https://github.com/aws-samples/aws-eks-accelerator-for-terraform.git`
+- Initialize a working directory:
+```
+cd examples/observability/eks-cluster-with-observability
+terraform init
+```
+- Fill-in the values for the variables in `dev.tfvars`
+  - The password for OpenSearch must be a minimum of eight characters with at least one uppercase, one lowercase, one digit, and one special character
+- Verify the resources created by this execution:
+```
+export AWS_REGION=<ENTER YOUR REGION>   # Select your own region
+terraform validate
+terraform plan -tf-vars=dev.tfvars
+```
+- Deploy resources with `terraform apply -tf-vars=dev.tfvars`
+- Add the cluster to your kubeconfig: `aws eks --region $AWS_REGION update-kubeconfig --name aws001-preprod-observability-eks`
+
+#### Map the FluentBit Role as a Backend Role to OpenSearch
+
+- Map the FluentBit Role as a Backend Role to OpenSearch:
```

Comment:
Lets explain what his is doing.

Reply:
How does this sound?

OpenSearch roles are the core method for controlling access to your OpenSearch cluster. Role mapping is part of OpenSearch's fine-grained access control security layer. Backend roles are a way to map an external identity to an OpenSearch role. In this case we map the FluentBit IAM role as a backend role to OpenSearch's *all_access* role so FluentBit can send logs to OpenSearch.